sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/action/ActionCreationContext.java

```diff
@@ -46,5 +46,8 @@ public interface ActionCreationContext extends Context {
    *
    * @return A tracer for the current action, if tracing is configured.
    */
+  @Deprecated(since = "1.4.2")
```

Comment:
In the java docs, we can add the following. Above the return.

```java
 * @deprecated use getTracer() instead.
 * 
 * @return A tracer for the current action, if tracing is configured.
```

Reply:
added